Services/COPage/classes/class.ilPageObject.php

```diff
@@ -2977,7 +2977,7 @@ public function deleteContent(string $a_hid, bool $a_update = true, string $a_pc
      * @return array|bool
      * @throws ilDateTimeException
      */
-    public function deleteContents(
+    public function deleteContents(// @TODO: PHP8 Review: Missing return type.
```

Comment:
@lscharmer Just a remark, there are multiple cases where missing return types are listed, but the return type is part of the phpdoc comment.